magpie/api/webhooks.py

```diff
@@ -35,6 +27,14 @@
 LOGGER = get_logger(__name__)
 
 
+class WebhookAction(ExtendedEnum):
+    """
+    Actions supported by webhooks.
+    """
+    WEBHOOK_CREATE_USER_ACTION = "create_user"
+    WEBHOOK_DELETE_USER_ACTION = "delete_user"
```

Comment:
Members should be only `CREATE_USER` and `DELETE_USER` since already within `WebhookAction` definition.